app/schemas/project_preference_update_settings_schema.rb

```diff
@@ -22,6 +22,11 @@ class ProjectPreferenceUpdateSettingsSchema < JsonSchema
         type "string", "integer"
         pattern "^[0-9]*$"
       end
+
+      property "designator" do
+        type "object"
+        additional_properties true
```

Comment:
So this will be free form object changing over time, or is it mostly set already?